ovssubnet/registry/registry.go

```diff
@@ -230,6 +238,11 @@ func (sub *EtcdSubnetRegistry) GetContainerNetwork() (string, error) {
 	return resp.Node.Value, err
 }
 
+func (sub *EtcdSubnetRegistry) GetServicesNetwork() (string, error) {
+	// FIXME
+	return "172.30.0.0/16", nil
```

Comment:
You can use existing 'servicesSubnet' master config option for this one.
pkg/cmd/server/api/v1/types.go: ServicesSubnet string `json:"servicesSubnet"`